.github/workflows/build_docker.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish Docker image
+
+on:
+  # push:
```

Comment:
any objections on building the container on each PR? We're quite happy with that on Vox Pupuli. We build a container for each merge on master/each pushed tag: https://github.com/voxpupuli/vox-pupuli-tasks/blob/master/.github/workflows/test-build-release.yml

Reply:
nope, just build it that way because i didn't know how often and when we want to build a container image 🤷‍♂️